internal/datastore/postgres/postgres.go

```diff
@@ -176,18 +182,30 @@ func NewPostgresDatastore(
 		UsersetBatchSize: int(config.splitAtUsersetCount),
 	}
 
+	quantizationPeriodNanos := int64(config.revisionQuantization)
+	if quantizationPeriodNanos < 1 {
+		quantizationPeriodNanos = 1
+	}
+	revisionQuery := fmt.Sprintf(
```

Comment:
Since most of the parameters are "static", could we somehow include them above and only parameterize on the `quantizationPeriodNanos`?

Reply:
This is only done once at initialization time.